_config.yml

```diff
@@ -113,12 +113,12 @@ projects:
     desc: Programming-by-demonstration tool for automating repetitive interactions with webpages.  Designed for non-programmers from the social sciences.
     img: /assets/images/projects/helena.png
     website: http://helena-lang.org/
-    displayorder: 4
+    displayorder: 5
```

Comment:
Hope the change in ordering I made is ok — let me know if you want me to just push this project to the end though!